micro-ui/web/micro-ui-internals/packages/modules/core/src/pages/employee/ProductDetails/ProductDetailsComponent.js

```
@@ -0,0 +1,98 @@
+import IconsObject from "./icons"; 
+
+const ProductDetailsComponent = ({ config }) => {
+  const { t } = useTranslation();
+  const { module } = useParams();
+  const history = useHistory();
+
+  const moduleConfig = config.find((item) => item.module === module) || {};
+  const IconComponent = moduleConfig.icon ? IconsObject[moduleConfig.icon] : null;
+
+  const handleButtonClick = (action) => {
+    const url = '/' + window.contextPath + action;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)